protocol/communities/community_changes.go

```diff
@@ -84,6 +88,22 @@ func (c *CommunityChanges) HasMemberLeft(identity string) bool {
 	return ok
 }
 
+func (c *CommunityChanges) HasMemberBanned(identity string) bool {
```

Comment:
I think you mean `IsMemberBanned`?

Reply:
You resolved this, but do you mean that this function is really to know if someone **has** banned someone else?

Reply:
I apologize, it seems I accidentally reverted it to `Has` back